docs/create-custom-image-using-existing-docker-image.md

```diff
@@ -115,19 +139,29 @@ alfresco-custom-image                            (Dir)
 * Create a Dockerfile for custom-alfresco-repository image:
 
 ```bash
-### Apply AMPs to the latest Enterprise repository image.
-FROM alfresco/alfresco-content-repository:latest
+# Apply AMPs to the latest Community repository image.
+ARG BASE_IMAGE=alfresco/alfresco-content-repository-community
+ARG VERSION=latest
+
+FROM ${BASE_IMAGE}:${VERSION}
+
+# Need these viariables to give permissions to tomcat (refer Dockerfile below)
+ARG GROUPNAME=Alfresco
+ARG USERNAME=alfresco
+ARG TOMCAT_DIR=/usr/local/tomcat
+
+# Need to switch to root to add the new files
```

Comment:
Please correct to:

Alfresco user does not have permissions to modify webapps or configuration. Switch to root.
The access will be fixed after all operations are done.